alpine-apache/README.md

```diff
@@ -0,0 +1,93 @@
+alpine-apache
+============
+
+An image for using [apache][apache], bundled with Alpine Linux and s6.
+
+**_Yet another container for running apache?_**
+
+Yes, but this one is built from [smebberson/alpine-base][alpinebase] that contains [s6][s6] for process management. Small, fast and with [s6][s6].
+
+_**Aren't you only supposed to run one process per container?**_
+
+Hell no! The following are good examples of when multiple processes within one container might be necessary:
+
+- Automatically updating [apache][apache] proxy settings when a down-stream application server (nodejs, php, etc) restarts (and the IP changes).
+- Automatically updating [HAProxy][haproxy] configuration to load balance to a group of down-stream application servers.
+- Running a logging daemon to centralize log management (i.e. [logentries][logentries], [loggly][loggly], [logstash][logstash]).
+- When you need to run a script on application server crash (to tidy something up), as the standard [Docker container restart policies][drsp] won't provide this.
+
+In all of these instances, there is one primary services and secondary support services. When the secondary support services fail, they should be automatically restarted. When the primary service fails, the container itself should restart.
+
+Versions
+--------
+
```

Comment:
Added. But I do think you need to fix it after accepting PR as the 1.0.0 link points at something right now that doesn't exist.